### PR TITLE
Schema: add acknowledgement_history

### LINKED ISSUE #19: Acknowledgements history
... with acks and ack clearings in addition to the notification history.

### PR DESCRIPTION
**Pull request by Al2Klimov**
fixes #19

### CODE REVIEW

File: etc/schema/mysql/mysql.schema.sql [hunk 1]
```diff
@@ -910,6 +910,24 @@ CREATE TABLE flapping_history (
...
+  event_time bigint(20) unsigned NOT NULL,
+  author varchar(255) NOT NULL COLLATE utf8mb4_unicode_ci,
+  comment text DEFAULT NULL,
+  expire_time bigint(20) unsigned DEFAULT NULL,
+  is_sticky enum('y','n') DEFAULT NULL,
+  is_persistent enum('y','n') DEFAULT NULL,
+
+  PRIMARY KEY (id)
+) ENGINE=InnoDb ROW_FORMAT=DYNAMIC DEFAULT CHARSET=utf8mb4 COLLATE utf8mb4_bin;
+
```
Comment (Al2Klimov) [inline etc/schema/mysql/mysql.schema.sql]: @lippserd Please have a look.

**Comment by lippserd** [comment]
While it makes sense to have both ack and user comments in the `comment` table, the ack comments should not appear in `comment_history`. Else we would have duplicate entries.

**Comment by lippserd** [comment]
Schema changes are good.

File: etc/schema/mysql/mysql.schema.sql [hunk 2]
```diff
@@ -922,8 +940,9 @@ CREATE TABLE history (
   downtime_history_id binary(20) NULL DEFAULT NULL COMMENT 'downtime_history.downtime_id',
   comment_history_id binary(20) NULL DEFAULT NULL COMMENT 'comment_history.comment_id',
   flapping_history_id binary(16) NULL DEFAULT NULL COMMENT 'flapping_history.id',
+  acknowledgement_history_id binary(16) NULL DEFAULT NULL COMMENT 'acknowledgement_history.id',
 
-  event_type enum('notification','state_change','downtime_schedule','downtime_start', 'downtime_end','comment_add','comment_remove','flapping_start','flapping_end') NOT NULL,
+  event_type enum('notification','state_change','downtime_schedule','downtime_start', 'downtime_end','comment_add','comment_remove','flapping_start','flapping_end','acknowledgement_set','acknowledgement_remove') NOT NULL,
```
Comment (N-o-X) [inline etc/schema/mysql/mysql.schema.sql]: Please rename `acknowledgement_set` and  `acknowledgement_remove` to `ack_set` and `ack_clear`

**Comment by N-o-X** [comment]
Please rebase

**Pull request merged**